src/game_actor.h

```diff
@@ -423,6 +423,13 @@ class Game_Actor : public Game_Battler {
 	 */
 	const std::vector<int16_t>& GetWholeEquipment() const;
 
+	/**
+	* Checks whether it has an item equipped.
```

Comment:
Suggestion: `Checks if the actor has a specific item equipped.`

Also, below: `Whether the item is equipped`